Trustroots/trustroots
# Migrate messages inbox to react

#### Proposed Changes

* this switches the inbox view  (`/messages`) to be a react component

### TODO

- [x] i18n everything in sight
- [x] add back `syncUnreadMessagesCount` broadcast
- [x] add back analytics call
- [x] remove legacy controller

#### Testing Instructions

* visit inbox, perhaps have a lot of threads so you see the "More messages" button
